apps/condo/package.json

```diff
@@ -107,7 +107,7 @@
     "immer": "9.0.13",
     "isomorphic-fetch": "^3.0.0",
     "js-cookie": "^2.2.1",
-    "jsonwebtoken": "^8.5.1",
+    "jsonwebtoken": "^9.0.0",
```

Comment:
Why it's need to be updated?

Reply:
v8.5.1 is of 2019 year, and there was no version dependant usages in code. In other apps it's version is already updates, so I decided to make it consistent. Also latest version has some security updates.